griptape/drivers/prompt/anthropic_prompt_driver.py

```diff
@@ -42,13 +48,19 @@ def default_prompt_stack_to_string_converter(self, prompt_stack: PromptStack) ->
 
         for i in prompt_stack.inputs:
             if i.is_assistant():
-                prompt_lines.append(f"Assistant: {i.content}")
-            else:
-                prompt_lines.append(f"Human: {i.content}")
+                prompt_lines.append(f"\n\nAssistant: {i.content}")
+            elif i.is_user():
+                prompt_lines.append(f"\n\nHuman: {i.content}")
+            elif i.is_system():
+                if self.model == "claude-2.1":
```

Comment:
Really don't love this, suggestions?

Reply:
One option would be to offload this to a prompt model driver for each version. Seems like overkill for a single switch like this but if the versions drift apart further it could reduce some possible logical complexity here.